### PR TITLE
Redesign of Header Toolbar

### PR DESCRIPTION
**Pull request by wes-cutting**
[Matching PR](https://seed-company-squad.monday.com/boards/3451697530/views/78801638/pulses/5592450753)

### CODE REVIEW

File: src/scenes/Root/Header/Header.tsx
Comment (CarsonF) [inline src/scenes/Root/Header/Header.tsx:1]: Let's change this to an [AppBar](https://mui.com/material-ui/react-app-bar/#app-bar-with-responsive-menu) to match new designs

File: src/scenes/Root/Header/ProfileMenu/ProfileMenu.tsx [hunk 1]
```diff
@@ -16,6 +24,9 @@ const useStyles = makeStyles()(({ spacing }) => ({
   menuHeading: {
     padding: spacing(1, 2, 2, 2),
   },
+  listItemText: {
+    textTransform: 'capitalize',
```
Comment (CarsonF) [inline src/scenes/Root/Header/ProfileMenu/ProfileMenu.tsx:36]: Why not just write the HTML as "Upload Manager"?
I get this idea for all caps styles, but this title case seems like normal writing..no?

File: src/scenes/Root/Header/ProfileToolbar/ProfileToolbar.tsx [hunk 2]
```diff
@@ -30,13 +23,13 @@ export const ProfileToolbar = () => {
...
-  const [actionsAnchor, setActionsAnchor] = useState<MenuProps['anchorEl']>();
 
   return (
     <>
-      <Card className={classes.card}>
+      <div className={classes.card}>
         <Typography className={classes.name} color="primary">
-          Hi, {session?.realFirstName.value ?? 'Friend'}
+          {session?.realFirstName.value ?? 'Friend'}{' '}
+          {session?.realLastName.value ?? ''}
```
Comment (CarsonF) [inline src/scenes/Root/Header/ProfileToolbar/ProfileToolbar.tsx:32]:
```suggestion
          {session.fullName}
```

File: src/scenes/Root/Header/ProfileToolbar/ProfileToolbar.tsx [hunk 3]
```diff
@@ -30,13 +23,13 @@ export const ProfileToolbar = () => {
   const { session } = useSession();
   const impersonation = useContext(ImpersonationContext);
   const [profileAnchor, setProfileAnchor] = useState<MenuProps['anchorEl']>();
-  const [actionsAnchor, setActionsAnchor] = useState<MenuProps['anchorEl']>();
 
   return (
     <>
-      <Card className={classes.card}>
+      <div className={classes.card}>
```
Comment (CarsonF) [inline src/scenes/Root/Header/ProfileToolbar/ProfileToolbar.tsx:29]: I'd change this to a horizontal stack spacing 1.

File: src/scenes/Root/Header/ProfileToolbar/ProfileToolbar.tsx [hunk 4]
```diff
@@ -30,13 +23,13 @@ export const ProfileToolbar = () => {
...
         <Typography className={classes.name} color="primary">
```
Comment (CarsonF) [inline src/scenes/Root/Header/ProfileToolbar/ProfileToolbar.tsx:30]:
```suggestion
        <Typography color="primary" sx={{ fontWeight: 'medium' }}>
```
And then the useStyles can be gotten rid of.

**Review by CarsonF — CHANGES_REQUESTED** [review]

File: src/scenes/Root/Header/ProfileToolbar/ProfileToolbar.tsx (hunk 3 above)
Comment (wes-cutting) [reply src/scenes/Root/Header/ProfileToolbar/ProfileToolbar.tsx:29]: <Stack direction="row" spacing={1}>

yields

Screenshot 2024-01-08 at 12 44 42

Comment (wes-cutting) [reply src/scenes/Root/Header/ProfileToolbar/ProfileToolbar.tsx:29]: Figured it out

File: src/scenes/Root/Header/Header.tsx
Comment (CarsonF) [reply src/scenes/Root/Header/Header.tsx:1]: Sorry what I meant here was switching to MUI `<AppBar>` component, not just rename our component.

Comment (wes-cutting) [reply src/scenes/Root/Header/Header.tsx:1]: Gotcha! On it

**Review by CarsonF — APPROVED** [review]

**Pull request merged**